src/main/java/model/TicketDAO.java

```diff
@@ -0,0 +1,163 @@
+package model;
```

Comment:
move TicketDAO and UserDAO to `dao` package, its not models

Reply:
Fixed. I moved DAO, TicketDAO and UserDAO into separated package